<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MG-APPS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="textEdit_FilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Select">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>3</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SelectFloder</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Configure</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>75</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TropDelay</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_TropDelay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>UNB3m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sasstam(GPT2)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hopfield(GPT2)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Method">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>50</y>
       <width>86</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Kalman</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SRIF</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>26</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cut-off Angle</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>433</x>
       <y>32</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kinematic</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Kinematic">
     <property name="geometry">
      <rect>
       <x>433</x>
       <y>52</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kin</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Angle">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>51</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>12</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Products</string>
     </property>
     <widget class="QRadioButton" name="radioButton_igs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>final</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cnt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>46</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>cnt</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox_PPPModel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Ion_free</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uncombined</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>333</x>
       <y>30</y>
       <width>75</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PPPModel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>626</x>
      <y>38</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Satellite system</string>
    </property>
    <widget class="QCheckBox" name="checkBox_GPS">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GPS</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_BDS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BDS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_GLONASS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>76</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GLONASS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_GAlieo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>105</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Galileo</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_Display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>771</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>136</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>PPP</string>
    </property>
    <widget class="QPushButton" name="pushButton_Run">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>89</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_RunBatch">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run(Batch)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Back">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>SPP</string>
    </property>
    <widget class="QPushButton" name="pushButton_SPP">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>25</y>
       <width>89</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_SPP_LC">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>86</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>P_IF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PL_IF</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_SPP_SMOOTH">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>25</y>
       <width>116</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>NoSmooth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Smooth</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>177</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Plot Stations:</string>
    </property>
    <widget class="QPushButton" name="pushButton_Plot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>63</y>
       <width>126</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_RunBatch">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>26</y>
       <width>126</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
